### ISSUE TITLE
Link to skip the toolbar

### ISSUE THREAD

**Issue by davidbenque** [cryptpad/cryptpad#1202]
> A skiplink should be provided to skip content that is repeated on multiple pages. In this case, it is a bit harder to decide whether the bar containing the logo and three buttons is ‘repeated content’ because of the title that differs on the pages. However, it would still be good for the usability and accessibility to provide a link that skips this bar. This link can be visually hidden and shown only when it receives the focus. It should be the first link on the page.
